### PR TITLE
tests/utils: fixed mode for write

### PR DESCRIPTION
**Pull request by jhscheer**
File was not write-able if opened with write.

### CODE REVIEW

**Comment by sylvestre** [comment]
what was the issue you were facing?

**Comment by jhscheer** [comment]
> what was the issue you were facing?

Creating a file with `at.write(<file>, <content>)` resulted in an empty file because it was opened in read-only mode.

File: tests/common/util.rs [hunk 1]
```diff
@@ -261,7 +261,10 @@ impl AtPath {
     }
 
     pub fn write(&self, name: &str, contents: &str) {
-        let mut f = self.open(name);
+        let mut f = OpenOptions::new()
```
Comment (Arcterus) [inline tests/common/util.rs:264]: This is missing a `log_info!()` statement.  Also, you can just use `File::create()` here instead.

Comment (jhscheer) [reply tests/common/util.rs:264]: I used touch to create and write to write.
`at.touch(<file>);
at.write(<file>, <content>)`
However, this resulted in an empty file.

**Comment by jhscheer** [comment]
A fix for this issue was recently merged with #1803
Maybe this can be closed. However #1803 is lacking the requested `log_info!()`

**Comment by sylvestre** [comment]
@jhscheer do you want to transform this one for log_info ?

**Comment by jhscheer** [comment]
> @jhscheer do you want to transform this one for log_info ?

Done. Should be good to merge.

**Pull request merged**